### DISCUSSION TITLE
[501940]Hikikomori No Chuunibyou

### DISCUSSION THREAD

**Discussion by nekoaweek** [pjy612/SteamManifestCache#575] [Share]
### Steam 游戏ID

501940

### 密钥列表

```hash
501941;37d28ae3c52ef678781182a8ed98362df16d166ff612e471aface13fab85884a
```

### 清单附件

[501941_6923227730959814057.zip](https://github.com/pjy612/SteamManifestCache/files/14949965/501941_6923227730959814057.zip)
[Hikikomori No Chuunibyou.zip](https://github.com/pjy612/SteamManifestCache/files/14950025/Hikikomori.No.Chuunibyou.zip)

### 备注

去年还是前年夏促，在用steampy阿根廷买错了的游戏，因为礼物没法退款
既然分享要的是库中没收录的，那么这算一个非常小众的（我也没玩）
附上游戏本体
今天看到release又开放了，之前还以为跑路了
这个邮箱算我小号，大号163实名不好公开，哪天国家想不开把github一抓，特地注册gamil本来是给ikun那个入库用的，结果删了。
email: nekoaweek@gmail.com
内测码:4C1CF40F1ED1E240(话说内测码现在有什么用？）

**Comment by pjy612** [comment]
不用上传本体。。。

**Reply by nekoaweek** [reply]
我看这本体比清单大不了多少，顺便就传了

**Reply by nekoaweek** [reply]
话说有没有所有收录的游戏appid名称集合。当初阿根廷买了一些乱七八糟的游戏，fc比较一下

另，这几天使用似乎发现sai一个bug，和仓库有关
